### PR TITLE
Add TiKV's mission

### PR DESCRIPTION
**Pull request by Hoverbear**
Add a mission statement to TiKV. @winkyao and @siddontang were part of the editing and revising of this mission.

### CODE REVIEW

File: text/2019-11-11-mission.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# TiKV's Mission
```
Comment (winkyao) [inline text/2019-11-11-mission.md]: Maybe TiKV's Vision?

Comment (Hoverbear) [reply text/2019-11-11-mission.md]: Okay! :)

Comment (Hoverbear) [reply text/2019-11-11-mission.md]: I thought about it and referred to places like https://www.bain.com/insights/management-tools-mission-and-vision-statements/ https://www.bain.com/insights/management-tools-mission-and-vision-statements/

And I think Mission is still best. Do you still prefer vision?

Comment (winkyao) [reply text/2019-11-11-mission.md]: "Vision Statements are often combined to provide a statement of the company’s purposes, goals and values. "

Comment (winkyao) [reply text/2019-11-11-mission.md]: I think "mission" is not suitable for a project. For this, I think maybe "vision" or "goal" is better.

File: text/2019-11-11-mission.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+> The CNCF fosters this landscape of open source projects by helping provide
+> end-user communities with viable options for building cloud native
+> applications. By encouraging projects to collaborate with each other, the CNCF
+> hopes to enable fully-fledged technology stacks comprised solely of CNCF member
+> projects. This is one way that organizations can own their destinies in the cloud.
+>
+> - https://www.cncf.io/blog/2018/11/05/34097/
+
+Our missions should, ultimately, echo that of the CNCF, but we did believe that
```
Comment (BusyJay) [inline text/2019-11-11-mission.md]: Why use past tense?

File: text/2019-11-11-mission.md (hunk 1 above)
Comment (Hoverbear) [reply text/2019-11-11-mission.md]: Well, I'm not going to choose between two terms I think are incorrect 🤣 , so please choose the one you and our maintainers can agree on and I will alter the RFC.

**Comment by Hoverbear** [comment]
Closing. I am not longer involved with the project and have no stake in this merging. Feel free to create a new PR.

**Pull request closed without merging**